generate.ts

```diff
@@ -42,8 +41,13 @@ var sdkGeneratorGlobals: SdkGenGlobals = {
     // Internal note: We lowercase the argsByName-keys, targetNames, buildIdentifier, and the flags.  Case is maintained for all other argsByName-values, and targets
     argsByName: {}, // Command line args compiled into KVP's
     errorMessages: [], // String list of errors during parsing and loading steps
-    targetOutputPathList: [], // A list of objects that describe sdk targets to build
-    buildFlags: [], // The sdkBuildFlags which modify the list of APIs available in this build of the SDK
+    buildTarget: {
+        buildFlags: [], // The flags applied to this build
+        destPath: null, // The path to the destination (usually a git repo)
+        srcFolder: null, // The SdkGenerator/targets/subfolder
+        versionKey: null, // The key in the API_Specs/SdkManualNotes.json file that has the version for this SDK
+        versionString: null, // The actual version string, from SdkManualNotes, or from another appropriate input
```

Comment:
>versionString [](start = 8, length = 13)

Do we need both a version string and a version key? seems like redundancy. Can we keep just the version value?

Reply:
I did this on purpose.
Customers will not be able to put their version numbers into our SdkManualNodes.json file, and for now, that file is extremely hard coded.
Customers will have to specifiy versionString in their genConfig.json.
However, I for now, I didn't want to break our SdkManualNodes.json flow.
I'm just thinking about more general usage and long term planning.